examples/gno.land/r/gnoland/gh/contract.gno

```diff
@@ -0,0 +1,105 @@
+package gh
```

Comment:
I suggest targeting a simpler API for these contracts. Any ideas for creating concise and user-friendly contract helpers?

Reply:
Have a look at the newer version of this. I've tried to distill down the boiler plate as much as I could.